### PR TITLE
MSchemaTool

### PR DESCRIPTION
**Pull request by dirkgr**
This converts papertasks to use MSchemaTool for schema migrations.

### CODE REVIEW

File: python/papertasks_test.py [hunk 1]
```diff
@@ -20,39 +22,55 @@ def setUp(self):
...
+
+        # Write an mschematool config, then read it.
+        with tempfile.NamedTemporaryFile(
+            mode="w+",
+            prefix="PapertasksMschema.",
+            suffix=".config",
+            encoding="UTF-8"
+        ) as configfile:
+            if self.db_user is not None:
+                connection_string = 'postgres://%s:%s@localhost/%s?options=-c%%20search_path%%3D%s' % (
```
Comment (dirkgr) [inline python/papertasks_test.py]: Turns out you can put the schema into the connection URL after all.

**Comment by dirkgr** [comment]
This is not actually ready. When I tried it, I ran the wrong thing.

File: python/papertasks_test.py (hunk 1 above)
Comment (jpowerwa) [reply python/papertasks_test.py]: :)

**Comment by jpowerwa** [comment]
Mod the test failure, of course.

**Comment by dirkgr** [comment]
Tests are passing. Yay!

**Pull request merged**